woocommerce/woocommerce-ios
# Shipping Labels: Remove ShippingLabelPackageDetails

Part of #6269 


### Description

`ShippingLabelPackageDetails` is the screen that handles configuring package in the shipping label creation flow. This screen has been replaced with `ShippingLabelPackagesForm` when supporting multiple packages, to handle a list of packages instead of a single package as before.

In #6270 the feature flag for multi-package support has been removed, so it's time to remove `ShippingLabelPackageDetails` entirely. There's a complication in this removal: we added some sample data for SwiftUI previews in `ShippingLabelPackageDetailsViewModel`, so removing this view model requires that the sample data is moved somewhere else. I created a new enum `ShippingLabelSampleData` to contain that code - feel free to suggest a better way to handle this if necessary.  

### Testing instructions

Please make sure that all package configuring functionalities still work as before:
1. Make sure that your test store has installed and configured WCShip plugin.
2. Create an order on your test store.
3. On the app, navigate to Orders > select the created order > select Create Shipping Label button to start the creation flow.
4. Configure Ship To and Ship From information for the label.
5. Proceed to Package - try moving packages, creating and selecting custom packages to make sure that everything still works.


---
